src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/ui/classbrowser/HTMLGenerator.java

```diff
@@ -883,10 +883,8 @@ protected String genPCHref(long targetPc) {
    }
 
    protected String genMultPCHref(String pcs) {
-      StringBuffer buf = new StringBuffer(genBaseHref());
-      buf.append("pc_multiple=");
-      buf.append(pcs);
-      return buf.toString();
+      String buf = genBaseHref() + "pc_multiple=" + pcs;
+      return buf;
```

Comment:
This is like VMVersionMismatchException.java above, where as it is reduced to two lines, we may as well reduce it to one line with:
return genBaseHref() + ..etc... 

With these two minor changes, I think this looks good.

Reply:
done

Reply:
Thanks - just a nit, looks like an extra space means "return genBaseHref() ..etc.." is not aligned.

Reply:
fixed